### DISCUSSION TITLE
Typos in the UI?

### DISCUSSION THREAD

**Discussion by ChrisNolan** [karrioapi/karrio#651] [Q&A]
I'm one of those 'grammar police' people you find on the internet occasionally ([ref xkcd 386](https://xkcd.com/386/)).  Do people have a preferred way for these types of changes to get PR'd (am I hip for trying to make that a verb?)?  For me I'd just change them when I found them and they'd end up bundled in whatever other change I was making.  Wasn't sure if you'd all rather have a single PR that was just those types of changes or not?

**Comment by danh91** [comment]
Yes please, a PR would be more than appreciated. 
Bad grammar included.

Side note: 😅 My first language is French. If you see anything at all that can be improved. Don’t hesitate to point out.
